src/main/dist/run_build.sh

```diff
@@ -84,7 +84,7 @@ echo ""
 if [[ -f "libs.r" ]]; then
     rm -f "libs.r"
 fi
-curl https://raw.githubusercontent.com/odysseusinc/DockerEnv/master/libs.r -o libs.r
+curl https://raw.githubusercontent.com/odysseusinc/DockerEnv/add-libgeos/libs.r -o libs.r
```

Comment:
why?

Reply:
Oh, thanks for review. It was assembly test track.